data/json/items/armor/suits_protection.json

```diff
@@ -844,6 +844,18 @@
     "looks_like": "touring_suit",
     "color": "dark_gray",
     "covers": [ "LEGS", "TORSO", "ARMS" ],
```

Comment:
I think these are obsolete now too.

Reply:
Right now they are not, however they absolutely should be made obsolete, it is more work that needs to be done.